driver/identity_service.py

```diff
@@ -6,12 +6,13 @@
 
 
 class NVMeshIdentityService(IdentityServicer):
-	def __init__(self, logger):
+	def __init__(self, logger, driver_name):
```

Comment:
I'm not sure the `NVMeshIdentityService` need the `driver_name` injected into the constructor, It probably should be the same as the Controller and NodeDriver and use `Config.DRVIER_NAME` directly in the constructor.

Reply:
Sorry for that, not much of a python dev...
That should be fixed now